front-end/sdk/InspectorBackend.js

```diff
@@ -676,6 +676,8 @@ InspectorBackendClass.WebSocketConnection = function(url, onConnectionReady)
  */
 InspectorBackendClass.WebSocketConnection.Create = function(url, onConnectionReady)
 {
+    var protocol = window.location.protocol === "https:" ? "wss:" : "ws:";
+    url = protocol + url.substr(3);
```

Comment:
I don't like magic number `3`.
Keep in mind that we update `front-end` folder automatically, the future webkit realisation can contain on this place `wss`, then result can be:
```js
// url == 'wss://127.0.0.1' => true
url = protocol + url.substr(3); // => wsss://127.0.0.1
```

Next code will be more safely:
```js
url = url.replace(/^ws:/, protocol);
```